.claude/skills/new-pr/SKILL.md

```
@@ -0,0 +1,26 @@
+description: 현재 브랜치에서 커밋한 내용을 바탕으로 새로운 pull request 를 하나 생성한다. pull request 본문 템플릿은 .github/PULL_REQUEST_TEMPLATE.md 를 사용한다.  create new pull request about current branch using .github/PULL_REQUEST_TEMPLATE.md
+---
+
+# new-pr
+
+새로운 pull request 를 생성한다.
+
+## 본문 형식
+다음의 경로에 있는 markdown 파일을 그대로 사용한다.
+
+```
+.github/PULL_REQUEST_TEMPLATE.md
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)